src/components/Graphs/Table/index.js

```diff
@@ -275,39 +275,58 @@ class Table extends AbstractGraph {
 
         if(!multiSelectable) {
             this.handleClick(...selectedRows)
+            this.selectedRows = {}
         }
 
         this.selectedRows[this.currentPage] = selectedRows.slice();
         this.setState({
             selected: this.selectedRows[this.currentPage]
         })
+
         const { selectRow, location } = this.props;
         if (selectRow) {
             let matchingRows = []
-            const selectedRows = this.getSelectedRows();
-            const row = selectedRows ? selectedRows[0] : {};
-
-            /**
-             * Compare `selectedColumn` value with all available datas and if equal to selected row,
-             * then save all matched records in store under "matchedRows",
-            **/
-
-            if(selectedColumn) {
-                const value = columnAccessor({column: selectedColumn})
-                matchingRows = data.filter( (d) => {
-                    return (value(row) || value(row) === 0) && row !== d && value(row) === value(d)
-                });
+
+            let row = {}
```

Comment:
maybe we need to call this rows!

Reply:
@natabal - I am not exactly sure about this. What do you mean by call here?

Reply:
@nxanil I mean the name of the variable should be 'rows' since the variable will potentially hold more then one row.